code/datums/status_effects/debuffs.dm

```diff
@@ -968,3 +968,53 @@
 /datum/status_effect/cloudstruck/Destroy()
 	. = ..()
 	QDEL_NULL(mob_overlay)
+
+/datum/status_effect/pranked
+	id = "pranked"
+	duration = 20 SECONDS
+	status_type = STATUS_EFFECT_REFRESH
+	alert_type = null
+	///increments every time the status effect is refreshed. Higher counter == more powerful clown combos.
```

Comment:
you should really document that the counter is turned into status effect duration

Reply:
...and the other stuff. i kinda wish you didn't do it like this like i said above, it's weird